### DISCUSSION TITLE
Exit Code 1 - Firefox

### DISCUSSION THREAD

**Discussion by ShadowDraft** [aclap-dev/video-downloadhelper#366] [Help and Q&A] [answered]
### Your question

Hello,
since yesterday I've had this error pop up:
```
Exit code: 1

moz-extension://9e3dc06f-131a-48e7-a9aa-1c09b739500a/background/main.js:1
receive@moz-extension://9e3dc06f-131a-48e7-a9aa-1c09b739500a/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://9e3dc06f-131a-48e7-a9aa-1c09b739500a/background/main.js:1:42626
```

Sometimes it works but stops the download at a seemingly random time.

I updated the CoApp and Addon. The other Exit Code 1 discussion I found had a different output so I hope it was correct to create a new discussion for mine.

Thank you for your help.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\sasor",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

Edit: And now it works again. I have no clue how or why though.

**Comment by Mattmjs** [comment]
Have you checked anti virus & firewall are not blocking the companion app.

**Reply by ShadowDraft** [reply]
Yes, neither are blocking. And I can start downloads seemingly random sometimes too. Some even finish while others get cut short.

**Reply by Mattmjs** [reply]
If you've got an example where this happens you should share it. If it's private then share the hit details information as text.

**Reply by ShadowDraft** [reply]
> If it's private then share the hit details information as text.

How do I do that?

**Reply by Mattmjs** [reply]
To the right of the downloads you'll see another button to click, then another menu opens. After that click details.
This shows more information about the video.

**Reply by ShadowDraft** [reply]
```
bitrate 5000000
bulk	undefined
chunked	hls
```
```java
core_media
{
    "builder": "Hls"
    "protocol": "hls"
    "duration": 7727.333333 "av": {
        "video": {
            "codec": {
                "name": "unknown"
                "type": "video"
                "mimetype": /.^/
                "defacto_container": "Mp4"
            }
            "bitrate": {
                "value": 5000000
            }
            "fps": {}
            "dimensions": {
                "value": {
                    "width": 1920 "height": 1080
                }
            }
            "quality": {
                "value": "1080"
            }
        }
        "audio": false
    }
    "container": {
        "name": "Mp4"
        "extension": "mp4"
        "audio_only_extension": "mp3"
        "defacto_codecs": {
            "audio": {}
            "video": {}
        }
        "supported_video_codecs": [
            0:
            "H264"
            1:
            "H265"
            2:
            "Av1"
            3:
            "MP4V"
            4:
            "MPEG2"
            5:
            "unknown"
        ]
        "supported_audio_codecs": [
            0:
            "Opus"
            1:
            "MP3"
            2:
            "FLAC"
            3:
            "AAC"
            4:
            "unknown"
        ]
        "mimetype": /(?:x-)?mp4/i
    }
}
```
```
created	1712130671313
descrPrefix HLS-Streaming
duration	7727.333333
extension	mp4
group	grp-3fa43fe
```
```java
headers
[
    0: {
        "name": "User-Agent"
        "value": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/201001 ..."
    }
    1: {
        "name": "Accept"
        "value": "*/*"
    }
    2: {
        "name": "Accept-Language"
        "value": "de,en-US;q=0.7,en;q=0.3"
    }
    3: {
        "name": "Accept-Encoding"
        "value": "gzip, deflate, br"
    }
    4: {
        "name": "Referer"
        "value": "https://missav.com/"
    }
    5: {
        "name": "Origin"
        "value": "https://missav.com"
    }
    6: {
        "name": "DNT"
        "value": "1"
    }
    7: {
        "name": "Sec-GPC"
        "value": "1"
    }
    8: {
        "name": "Connection"
        "value": "keep-alive"
    }
    9: {
        "name": "Sec-Fetch-Dest"
        "value": "empty"
    }
    10: {
        "name": "Sec-Fetch-Mode"
        "value": "cors"
    }
    11: {
        "name": "Sec-Fetch-Site"
        "value": "cross-site"
    }
]
```
```
id	hls:2a567a4c
isPrivate true
masterManifest	https://surrit.com/cc7b93cb-4b32-4e85-bfdd-54d73dfe7cf4/playlist.m3u8
mediaManifest	https://surrit.com/cc7b93cb-4b32-4e85-bfdd-54d73dfe7cf4/1920x1080/video.m3u8
originalId	network-probe:7e91c1f3
pageUrl	https://missav.com/en/pft-001-uncensored-leak
referrer	https://missav.com/en/pft-001-uncensored-leak
size	1920x1080
status	active
tabId	93
thumbnailUrl	https://eightcha.com/pft-001-uncensored-leak/cover.jpg?class=normal
title	PFT-001 Mankitsuchu Original work by Haruto Torasuke A
topUrl	https://missav.com/en/pft-001-uncensored-leak
```

**Reply by Mattmjs** [reply, marked as answer]
Ok you expanded everything which wasn't necessary and the video wasn't private. I only ask for this information if the video is
private meaning it is behind a login & password. This one isn't, I could access it. I tested it and got the exit code error in the stable version. I tested in beta extension & it works. So wait for an updated stable version or install the beta now.
